frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "@angular/router": "~10.0.5",
     "@ng-bootstrap/ng-bootstrap": "^7.0.0",
     "bootstrap": "^4.5.0",
+    "ngx-bootstrap": "^5.6.1",
```

Comment:
Could you use added earlier @ng-bootstrap instead of ngx-bootstrap? They are pretty similar, so it won't difficult to change.

Reply:
ok sure